addons/binding/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/handler/ChromecastHandler.java

```diff
@@ -187,17 +204,36 @@ private void handlePlayUri(Command command) {
 
     private void handleControl(final Command command) {
         try {
+            if (chromecast.getRunningApp() == null) {
+                logger.warn("{} command ignored because media player app is not running", command);
+                return;
+            }
+
+            if (command instanceof NextPreviousType) {
+                logger.warn("{} command not yet implemented", command);
+                return;
+            }
+
             if (command instanceof PlayPauseType) {
+                MediaStatus mediaStatus = chromecast.getMediaStatus();
+                logger.debug("mediaStatus {}", mediaStatus);
+                if (mediaStatus == null || mediaStatus.playerState == PlayerState.IDLE) {
+                    logger.warn("{} command ignored because media is not loaded", command);
+                    return;
+                }
+
                 final PlayPauseType playPause = (PlayPauseType) command;
                 if (playPause == PlayPauseType.PLAY) {
                     chromecast.play();
-                } else {
+                } else if (playPause == PlayPauseType.PAUSE
+                        && ((mediaStatus.supportedMediaCommands & 0x00000001) == 0x1)) {
                     chromecast.pause();
+                } else {
+                    logger.warn("{} command not supported by current media", command);
                 }
             }
-            updateStatus(ThingStatus.ONLINE);
-        } catch (final IOException e) {
-            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR, e.getMessage());
```

Comment:
Because if a play/pause command failed, it does not mean that the thing is OFFLINE.
The problem was that if it occurs, the thing remained OFFLINE until you restarted OH. Not satisfying.